File: Winform/FaceRecognize/Medicine.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace FaceRecognize
{
    public class Medicine
    {
        public string name { get; set; }
        public String session { get; set; }
        public int dose { get; set; }
        public int numOfDays { get; set; }
        public int total { get; set; }

        public Medicine(String name, String session, int dose, int numofday, int total) { 
            this.name = name;  
            this.session = session;
            this.dose = dose;
            this.numOfDays = numofday;
            this.total = total;
           
        }
    }
}

File: Android/app/src/main/java/com/example/facerecognition/Class/Medicine.java
package com.example.facerecognition.Class;

import java.util.ArrayList;
import java.util.List;

public class Medicine {
    public String name;
    public int dose;
    public int numOfDays;
    public String session;
    public int total;

    public Medicine() {
    }

    public Medicine(String name, int dose, int numOfDays, String session, int total) {
        this.name = name;
        this.dose = dose;
        this.numOfDays = numOfDays;
        this.session = session;
        this.total = total;
    }

    public Medicine(String name, String dose, String numOfDays, String session, String total) {
        this.name = name;
        this.dose = Integer.parseInt(dose);
        this.numOfDays = Integer.parseInt(numOfDays);
        this.session = session;
        this.total = Integer.parseInt(total);
    }
    public static List<Medicine> medicine_List_static = new ArrayList<>();
}
